fix: ensure tRBTC faucet GA event fires on external link click (#498)
rsksmart/devportal: _rsk/rbtc/conversion/with-trezor-t.md

---
layout: rsk
title: Accessing and using funds that are not in accounts derived with RSK dpath in Trezor T
tags: rsk, rbtc, conversion, peg, 2-way, peg-in, peg-out, federation, trezor, dpath
description: 'How to configure a Trezor T hardware wallet to derive with a custom dpath'
collection_order: 3160
---

> How to solve the problem of moving your funds when they are in a account that needs to

Reviewer: ```suggestion
How to solve the problem of moving your funds when they are in an account that needs to
```

be derived with a custom derivation path (dpath) using Trezor T.

## General context

If you made a [BTC to R-BTC conversion](#btc-to-r-btc-conversion) using Trezor T, you need to access your account by using a custom dpath (`44'/0'/0'/0/0` for Mainnet). With the last firmware versions, Trezor T is checking that the dpath matches with the expected one as a safety feature and this is a blocker when you intend to use a different dpath.
You may also want to access your account with a different dpath if you made a mistake; for example, receiving RBTC at an address derived using the Ethereum dpath instead of the RSK dpath.

In MyCrypto or MyEtherWallet you may have received this message: `"Forbidden key path"`.

## Solution

To allow custom derivation paths, you will need to turn off safety checks (see [Pavol Rusnak message](https://github.com/trezor/trezor-firmware/issues/1255#issuecomment-691463540)).

> To do this, you need first to install [python-trezor](https://github.com/trezor/python-trezor):

Reviewer: ```suggestion
To do this, you need to install [python-trezor](https://github.com/trezor/python-trezor):
```

---
layout: rsk
title: Accessing and using funds that are not in accounts derived with RSK dpath in Trezor T
tags: rsk, rbtc, conversion, peg, 2-way, peg-in, peg-out, federation, trezor, dpath
description: 'How to configure a Trezor T hardware wallet to derive with a custom dpath'
collection_order: 3160
---

> This section is meant to explain how to solve the problem of moving your funds when they are in a account that needs to
> be derived with a different derivation path using Trezor T.

Reviewer 2: ```suggestion
How to solve the problem of moving your funds when they are in a account that needs to
be derived with a custom derivation path (dpath) using Trezor T.
```

## General context

> If you made a [BTC to R-BTC conversion](#btc-to-r-btc-conversion) using Trezor T, you need to access your account by using a custom dpath (`44'/0'/0'/0/0` for Mainnet). With the last firmware versions, Trezor T is checking that the derivation path matches with the expected one as a safety feature and this is a blocker when you need to consciously use a different dpath.

Reviewer 2: ```suggestion
If you made a [BTC to R-BTC conversion](#btc-to-r-btc-conversion) using Trezor T, you need to access your account by using a custom dpath (`44'/0'/0'/0/0` for Mainnet). With the last firmware versions, Trezor T is checking that the dpath matches with the expected one as a safety feature and this is a blocker when you intend to use a different dpath.
```

> You may also want to access your account with a different dpath if you made a mistake like receiving RBTC in an Ethereum address.

Reviewer 2: ```suggestion
You may also want to access your account with a different dpath if you made a mistake; for example, receiving RBTC at an address derived using the Ethereum dpath instead of the RSK dpath.
```

> You probably tried to do it in MyCrypto or MyEtherWallet and you got this message: `"Forbidden key path"`.

Reviewer 2: ```suggestion
In MyCrypto or MyEtherWallet you may have received this message: `"Forbidden key path"`.
```
